techlibs/xilinx/synth_xilinx.cc

```diff
@@ -306,6 +306,10 @@ struct SynthXilinxPass : public ScriptPass
 			run("proc");
 			if (flatten || help_mode)
 				run("flatten", "(with '-flatten')");
+			active_design->scratchpad_unset("tribuf.added_something");
+			run("tribuf -logic");
+			if (!do_iopad && active_design->scratchpad_get_bool("tribuf.added_something"))
```

Comment:
This should probably be `if (no_iopad && ...`? `do_iopad` doesn't seem to be defined anywhere.

Reply:
Resolved in 245b8c4a. Thanks!